integration/apidefinition_test.go

```diff
@@ -32,6 +32,11 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
+var (
```

Comment:
these should be const

Reply:
the returned value from `fmt.Errorf` or `errors.New` are not feasible to assign constant values.

Reply:
you can return them as "constantVar + %v"